embedded-graphics/src/primitives/rectangle.rs

```diff
@@ -107,6 +106,19 @@ impl Rectangle {
 
         self.top_left + Size::new(dx, dy)
     }
+
+    /// Returns the bottom right corner of this rectangle.
+    ///
+    /// Because the smallest rectangle that can be represented by its corners
+    /// has a size of 1 x 1 pixel this functions returns `None` if the width or
+    /// height of the rectangle is zero.
```

Comment:
```suggestion
    /// Because the smallest rectangle that can be represented by its corners
    /// has a size of 1 x 1 pixels, this function returns `None` if the width or
    /// height of the rectangle is zero.
```